### PR TITLE
Add advanced wardrobe keybinds

### PR DESCRIPTION
**Pull request by RayDeeUx**
introducing:
- keybinds to advance and go back!
- keybinds to unequip your current wardrobe slot!

### CODE REVIEW

File: src/main/java/io/github/moulberry/notenoughupdates/miscfeatures/WardrobeMouseButtons.kt [hunk 1]
```diff
@@ -57,11 +64,41 @@ class WardrobeMouseButtons {
...
+                            else {
+                                notEquipped++
+                            }
+                        }
+                        if (notEquipped == 9) return
+                    } else {
+                        slotNum = (36 + i)
+                    }
+                    if (container.getSlot(slotNum).getStack().getDisplayName().isEmpty()) return
+                    if (slotNum < 36 || ((slotNum > 45) && (slotNum != 53))) return
```
Comment (romangraef) [inline src/main/java/io/github/moulberry/notenoughupdates/miscfeatures/WardrobeMouseButtons.kt:100]: I don't think this belongs inside of the for loop?

**Review by romangraef — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/moulberry/notenoughupdates/miscfeatures/WardrobeMouseButtons.kt (hunk 1 above)
Comment (RayDeeUx) [reply src/main/java/io/github/moulberry/notenoughupdates/miscfeatures/WardrobeMouseButtons.kt]: whoops! rewrote

**Review by romangraef — APPROVED** [review]

**Review by NopoTheGamer — CHANGES_REQUESTED** [review]
crashes: https://hst.sh/raw/rumayuceto

**Comment by RayDeeUx** [comment]
> crashes: https://hst.sh/raw/rumayuceto

i think i fixed it with my latest commits just a few moments ago @NopoTheGamer please test thank you

**Review by romangraef — DISMISSED** [review]

**Comment by RayDeeUx** [comment]
update: i think in the process of fixing nopo's NPE i got another NPE of my own also i have no idea what triggers it so give me a week or two, if i dont report back with another NPE then yall can decide to merge

**Comment by RayDeeUx** [comment]
> update: i think in the process of fixing nopo's NPE i got another NPE of my own also i have no idea what triggers it so give me a week or two, if i dont report back with another NPE then yall can decide to merge

image

**Comment by RayDeeUx** [comment]
ok final update: caught hold of both NPEs, hoepfully no more crop up during the next two weeks

**Review by romangraef — APPROVED** [review]

**Comment by RayDeeUx** [comment]
ok fuck nvm it won't compile back to square one

**Review by NopoTheGamer — APPROVED** [review]
doesnt crash !

**Comment by RayDeeUx** [comment]
@romangraef nopo has confirmed that all NPEs are gone, gg

**Review by romangraef — APPROVED** [review]
I finally figured out how to repro the crash and now it's fixed :(

**Pull request merged**